core/templates/local_vector.h

```diff
@@ -178,7 +178,8 @@ class LocalVector {
 	}
 
 	int64_t find(const T &p_val, U p_from = 0) const {
-		for (U i = 0; i < count; i++) {
+		ERR_FAIL_INDEX_V(p_from, count, -1);
```

Comment:
Shouldn't it be `ERR_FAIL_UNSIGNED_INDEX_V`  like for other checks in this class? I'm surprised that it's not raising a warning about tautological comparison to 0 though.

Reply:
U is default defined as unsigned, but I don't see how it's enforced to be unsigned if it's defined signed specifically :S

Reply:
But I'll change it, it does not make sense to make this check only on one of the functions, though I'd expect the compilers to actually optimize that check out in the cases in which it is tautological.